cli/src/main/java/com/devonfw/tools/ide/context/GitContextImpl.java

```diff
@@ -104,7 +97,7 @@ public void pullOrClone(String gitRepoUrl, String branch, Path targetRepository)
 
     if (Files.isDirectory(targetRepository.resolve(".git"))) {
       // checks for remotes
-      ProcessResult result = this.processContext.addArg("remote").run(PROCESS_MODE);
+      ProcessResult result = this.processContext.addArg("remote").run(ProcessMode.DEFAULT_CAPTURE);
```

Comment:
To fix the error with git remote check returning error code 128:
```suggestion
      this.processContext.directory(targetRepository);
      ProcessResult result = this.processContext.addArg("remote").run(ProcessMode.DEFAULT_CAPTURE);
```
Looks like I've missed this in my last PR.